spec/main-process/file-recovery-service.test.js

```diff
@@ -112,7 +112,7 @@ describe("FileRecoveryService", () => {
       const mockWindow = {}
       const filePath = temp.path()
       fs.writeFileSync(filePath, "content")
-      fs.chmodSync(filePath, 0444)
+      fs.chmodSync(filePath, 0o444)
```

Comment:
Yeah, octal literals w/ leading zeros was a non-standard extension that some browsers provided, but I guess the new babel doesn't allow.